.github/workflows/bft-integration-tests.yml

```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
+
+name: bft-integration-tests
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the release-1.4-BFT-3 branch
```

Comment:
can we please also align the comments to the right branches? there is no branch release-1.4-BFT-3 here